src/main/java/org/assertj/core/api/AbstractThrowableAssert.java

```diff
@@ -203,6 +203,31 @@ public SELF hasNoCause() {
     return new ThrowableAssert<>(org.assertj.core.util.Throwables.getRootCause(actual));
   }
 
+  /**
+   * Returns a new list assertion of every cause in the chain of causes.
+   * The list is ordered from the current {@link Throwable} up to the root cause.
```

Comment:
Myself, I kinda prefer the term `down to the root cause`.

Reply:
`Returns a new list assertion of every cause down to the root cause. The list is empty if there is no cause.`  sounds good to me